kitsune/settings.py

```diff
@@ -757,6 +758,16 @@ def JINJA_CONFIG():
 # Default storage engine - ours does not preserve filenames
 DEFAULT_FILE_STORAGE = 'kitsune.upload.storage.RenameFileStorage'
 
+# AWS S3 Storage Settings
+# local_settings must include the following to use s3 storage
+# AWS_ACCESS_KEY_ID
+# AWS_SECRET_ACCESS_KEY
+# AWS_STORAGE_BUCKET_NAME
+AWS_S3_OBJECT_PARAMETERS = {
+    'CacheControl': 'max-age=86400',
+}
+AWS_S3_CUSTOM_DOMAIN = 'support.cdn.mozilla.net'
```

Comment:
Let's change this to the production cdn for user media `prod-cdn.sumo.mozilla.net`.

If we decide to push static to s3 we will need to create 2 different storage classes to set two different custom domains.